docs/src/requirements/child-devices.md

```diff
@@ -15,18 +16,64 @@
 
 TODO: do not forget, a child-device could be also logical (a process running on the gateway)
 
+# Initialisation of a Child-Device
+Before a child-device could be put into action, both aspects from above (_Device-Side Endpoint_ and _Cloud Child Device-Twin_) need to be initialised.
+
+* Initialisation: Device-Side Endpoint
+
+   ...TODO...
+
+* Initialisation: Cloud Child Device-Twin
+
+  Two levels of initialisations have to take place for the cloud child device-twin: 
+  1) creating the new child-device twin in the cloud
+  2) declaring capabilities on the new child-device twin, i.E.:
+     - declaring supported operations (e.g. sw-update, config-man, log-man, ...)
+     - declaring supported types (e.g. sw module-type, config-type, log-type, ...)
+
 # User Journey
 
-## Connecting a smart physical child-device (featuring MQTT client) to the cloud via the gateway
-- User creates the child-device twin in the cloud, with a unique child-id and supported operations
-- User plugs physical child-device to the gateway
-- User configures the broker IP/Port/Certificate and child-id in the physical child-device
-- User configures the root certificate in the gateway
-- Physical child-device asks thin-edge to validate if cloud child-device twin with the same child-id exists
-- Gateway consults cloud to confirm if cloud child-device twin with child-id exists
-- Result of the validation is reported to the physical child-device
-- If the validation succeeds, the physical child-device starts sending/receiving data/operations
-- If the validation fails, the physical child-device can either create the cloud child-devive twin, or repeat the validation until a 3rd party creates the cloud child-device twin, or signal an error to the user.
+## Definition of Personas used in the user journeys below
+* __Device Operator__ (the one who maintains/operates the device in the field in the shopfloor)
+* __Agent Developer__ (the one who adapts and deploys thin-edge to the device)
+* __Device App Developer__ (the one who implements the device application that makes use of thin-edge APIs)
+* __Solution Architect__ (the one, who is in authority for the whole end-to-end picture (from devices to cloud) )
```

Comment:
I would simply ignore this role. Thin-edge provides nothing for these architects beyond APIs and tools to be respectively used by agent developers and device operators.